cfme/configure/access_control.py

```diff
@@ -1,61 +1,116 @@
-from functools import partial
-
 from navmazing import NavigateToSibling, NavigateToAttribute
+from widgetastic_manageiq import UpDownSelect, SummaryFormItem
+from widgetastic_patternfly import (
+    BootstrapSelect, Button, Input, Tab, CheckableBootstrapTreeview,
+    BootstrapSwitch, CandidateNotFound)
+from widgetastic.widget import Text, Checkbox, View
 
-import cfme.fixtures.pytest_selenium as sel
-import cfme.web_ui.toolbar as tb
 from cfme.base.credential import Credential
-from cfme.exceptions import CandidateNotFound, OptionNotAvailable
-from cfme.web_ui import (
-    AngularSelect, Form, Select, CheckboxTree, accordion, fill, flash,
-    form_buttons, Input, Table, UpDownSelect, CFMECheckbox, BootstrapTreeview)
+import cfme.fixtures.pytest_selenium as sel
+from cfme.exceptions import OptionNotAvailable
+from cfme.web_ui import Table
```

Comment:
Currently, I left the old table, as the new widgetastic table does not cover all needed functionality for tag_table(ex. delete tag button). As soon as widget will be added or updated I will replace it with widgetastic.